rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/model/EjbSessionBeanModel.java

```diff
@@ -80,11 +80,11 @@
      * Contains the jndi location for this resource.
      */
     @Adjacency(label = JNDIResourceModel.TYPE, direction = Direction.OUT)
-    public JNDIResourceModel getJndiReference();
+    public Iterable<JNDIResourceModel> getJndiReferences();
 
     /**
      * Contains the jndi location for this resource.
      */
     @Adjacency(label = JNDIResourceModel.TYPE, direction = Direction.OUT)
-    public void setJndiReference(JNDIResourceModel jndiReference);
+    public void addJndiReference(JNDIResourceModel jndiReference);
```

Comment:
@mbriskar I think we should just start with the Global, Module, and App references.  If we find other JNDI references for EJBs in the future, we could consider expanding beyond, but think an additional model might not be worth the added complexity at this point.